project_department/__manifest__.py

```diff
@@ -2,8 +2,8 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Project Department Categorization",
-    "version": "11.0.1.0.0",
-    "author": "Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "QubiQ, Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
```

Comment:
Porting a module is not enough to grant authorship.
Please add yourself to the contributors list instead.